### ISSUE TITLE
.vsc.attach() not working as expected if httpgd is enabled (on remote tmux)

### ISSUE THREAD

**Issue by CyrilLagger** [REditorSupport/vscode-R#869]
Hi, thank you for the great extension!

**Describe the bug**

I am using vscode-R to work on a Docker container running on a remote server I access via SSH. I am working with tmux to preserve my sessions. I am experiencing problems with `.vsc.attach()` when I enable httpgd, i.e. when adding `options(vsc.use_httpgd = TRUE)` to my `~/.Rprofile`. 

Let's say I have two terminals open in tmux, each running an R session with different environments. When switching from one terminal to another and running `.vsc.attach()`, the expected behaviour is to see the vscode-R workspace being updated to display the correct environment variables. This works fine for me when httpgd is disabled.

However, when httpgd is enabled, the workspace panel does not update IF a graphic command has been previously run from the current terminal. What `.vsc.attach()` does, in that case, is updating the vscode-R graphic panel, but not the workspace panel. 

The same thing happens if I work with a single terminal in tmux, but close and reopen vscode. I won't be able to get the workspace panel to display anything anymore. But no problem when httpgd is disabled. 

I am happy to test a few suggestions to solve this, but I currently don't know what to do about it.

**Screenshots**
Step-by-step process to reproduce the problem:

![bug_vscR](https://user-images.githubusercontent.com/56308637/142212969-2661d394-f41d-4720-aef3-2e56a346d447.PNG)

**Environment (please complete the following information):**

- OS: Linux-Docker container running on a Linux remote server access via SSH from Windows
- VSCode Version: 1.62.2
- R Version: 4.1.1
- vscode-R version: 2.3.2

**Additional context**

My `~/.Rprofile` looks like this:
```
Sys.setenv(TERM_PROGRAM="vscode")
source(file.path(Sys.getenv(if (.Platform$OS.type == "windows") "USERPROFILE" else "HOME"), ".vscode-R", "init.R"))
options(vsc.use_httpgd = TRUE)
```

**Comment by renkun-ken** [comment]
Duplicate of #832 and fixed via #852.

You might want to try the [latest development version](https://github.com/REditorSupport/vscode-R/releases/tag/latest) which includes the fix.

**Comment by CyrilLagger** [comment]
Thanks!

I installed the latest development version and everything works fine. Very happy to now have access to httpgd while using tmux remotely.
